package.json

```diff
@@ -109,6 +109,7 @@
     "sequelize-cli": "^6.3.0",
     "simple-git": "^3.6.0",
     "sleep-async": "^1.0.5",
+    "sparqljs": "^3.5.1",
```

Comment:
Why did you add this package?

Reply:
I remove it from package.json file. I installed when I have issue with query scenario.